greenplumpython/core/gp_connection.py

```diff
@@ -16,18 +16,21 @@ def _sqlalchemy_connection_convert(self, conn):
         return conn_alchemy
 
     def connect(self, host: str, port: int, database: str, user: str, password: str):
-        url = "postgresql+pygresql://%s:%s@%s:%d/%s" % (user, password, host, port, database)
+        url = "postgresql+psycopg2://%s:%s@%s:%d/%s" % (user, password, host, port, database)
         engine = sqlalchemy.create_engine(url)
         conn = engine.connect()
-        conn_alchemy = self._sqlalchemy_connection_convert(conn)
-        conn_pairs = (conn, conn_alchemy)
-        self.connection_pool[self.max_connectid] = conn_pairs
-        self.max_connectid += 1 
-        return conn_alchemy
-    
+        #conn_alchemy = self._sqlalchemy_connection_convert(conn)
+        #conn_pairs = (conn, conn_alchemy)
+        #self.connection_pool[self.max_connectid] = conn_pairs
```

Comment:
when merge we can delete these comments

Reply:
Yes, I didn't delete them in case they do something special that we didn't identify yet.